articles/iot-hub-device-update/create-update-group.md

```diff
@@ -197,6 +197,18 @@ az iot du device group show \
 
 ---
 
+## Removing a device from a device group
+
+To remove a device from a device group, the ADU Group tag value must be changed to "null".
+
+   ```JSON
+       "tags": {
+               "ADUGroup": "null"
+               }
+   ```
+This will delete the ADUGroup tag from the device twin and remove the device from its device group.
+
+---
```

Comment:
```suggestion
To remove a device from a device group, the `ADUGroup` tag value must be changed to "null".

   ```JSON
       "tags": {
               "ADUGroup": "null"
               }
   ```
This will delete the `ADUGroup` tag from the device twin and remove the device from its device group.

```